pom.xml

```diff
@@ -229,12 +233,24 @@
 				<artifactId>junit</artifactId>
 				<version>${junit.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>org.junit</groupId>
+				<artifactId>junit-bom</artifactId>
+				<version>${junit-jupiter.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
 			<dependency>
 				<groupId>com.avanza.gs</groupId>
 				<artifactId>gs-test</artifactId>
 				<version>${gs-test.version}</version>
 				<scope>test</scope>
 			</dependency>
+			<dependency>
+				<groupId>com.tngtech.archunit</groupId>
+				<artifactId>archunit-junit5</artifactId>
+				<version>${archunit.version}</version>
+			</dependency>
```

Comment:
I don't see ArchUnit being used, to I think the dependency could be removed.

Reply:
Fixed!